src/app.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+async function getLand(land) {
+    try {
+        const uri = "https://restcountries.com/v2/name/" + land;
```

Comment:
Ik vind het altijd iets netter (en beter leesbaar als je meer dingen aan elkaar gaat plakken) als je het zo doet
```suggestion
        const uri = `https://restcountries.com/v2/name/${land}`;
```

Reply:
Ooo, bedankt. Ik probeert dat precies te doen, maar het werkte niet en nu weet ik, dat het was vanwege het aanhalingsteken!